shared/src/address/payload.rs

```diff
@@ -19,6 +19,20 @@ pub struct DelegatedAddress {
     buffer: [u8; MAX_SUBADDRESS_LEN],
 }
 
+#[cfg(feature = "arb")]
+impl quickcheck::Arbitrary for DelegatedAddress {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        Self {
+            namespace: ActorID::arbitrary(g),
+            length: usize::arbitrary(g),
+            buffer: arbitrary::Arbitrary::arbitrary(&mut arbitrary::Unstructured::new(
+                &Vec::arbitrary(g),
+            ))
```

Comment:
Avoid using the `arbitrary` crate. Everything should be doable using only `quickcheck`.

Reply:
I havent found a way to nicely create a [u8; 20] with quickcheck. Just using the arbitrary implementation we already had in Forest. Any recommendations would be nice.